src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -129,6 +217,31 @@
                 $('#aviationTimeConfig').removeClass('hidden');
             }
         });
+
+        function confirmChoice(msg, button, callback) {
+            $.confirm({
+                text: msg,
+                title: button,
+                cancelButton: "No",
+                confirm: function() {
+                    callback();
+                },
+
+                cancel: function(){
+                    location.reload();
+                }
+            });
+        }
+
+        function hasAssignmentConfiguration() {
+            if($('#assignmentName') == null || $('#assignmentPoints') == null || $('#assignmentName') == '' || $('#assignmentPoints') == '') {
```

Comment:
Jquery selectors do not return null if no value is found. They do have a length property that you can check though.

Also just a heads up that you might want to look at this since javascript equality is super confusing https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness